redpanda/validators/validators.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Redpanda Data, Inc.
```

Comment:
[nit] :sweat_smile: 
```suggestion
// Copyright 2024 Redpanda Data, Inc.
```

Reply:
Time is immaterial to the redpanda, there is only the eternal now 🧋